### ISSUE TITLE
Plugin Not Activating with wpcom_vip_load_plugin

### ISSUE THREAD

**Issue by kala725** [Automattic/Edit-Flow#368]
Plugin Not Activating with wpcom_vip_load_plugin instead I have to activate from plugins admin screen. How can I achieve the same?

**Comment by david-binda** [comment]
Hello @kala725 ! Could you, please, provide more detailed information on what's broken? Reproduction steps with code examples would be awesome. Thanks!

**Comment by cojennin** [comment]
@kala725 just checking back in. Any updates, more information or steps to reproduce you can provide about what's broken?

**Comment by kala725** [comment]
@cojennin : I have to activate the plugin using the activate_plugin function call. Ideally it should load with wpcom_vip_load_plugin , if it is not dependent on the plugins_loaded hook

**Comment by david-binda** [comment]
Hello @kala725 ! How specifically are you using the `wpcom_vip_load_plugin()` function? Could you, please, share the code with us?

One possible cause might be the fact that this GitHub repository is not named the same way as the default plugin's file (note the `Edit-Flow` vs. `edit_flow.php`).

Calling the `wpcom_vip_load_plugin( 'edit-flow' );` would then check for existence of following files and fails (see related code: https://github.com/Automattic/vip-go-mu-plugins/blob/41dfb83d11d1b6605450f0dcf4791abb5aa9414f/vip-helpers/vip-utils.php#L1003,L1027 ):

```
wp-content/plugins/edit-flow/edit-flow.php
wp-content/plugins/edit-flow/plugin.php
wp-content/mu-plugins/shared-plugins/edit-flow/edit-flow.php
wp-content/mu-plugins/shared-plugins/edit-flow/plugin.php
```

Thus you either must checkout this repository to a different folder:

```
git clone https://github.com/Automattic/Edit-Flow edit_flow
```

or specify a full file path in the `wpcom_vip_load_plugin` call:

```
wpcom_vip_load_plugin( 'edit-flow/edit_flow.php' )
```
(related code: https://github.com/Automattic/vip-go-mu-plugins/blob/41dfb83d11d1b6605450f0dcf4791abb5aa9414f/vip-helpers/vip-utils.php#L978,L984 )

I also guess that this is directly related to your WordPress.com VIP work. If that's the case, the best and fastest path for getting our support is to open a Zendesk ticket with us.

**Comment by kala725** [comment]
Hi @david-binda , I am using the following command 

`wpcom_vip_load_plugin( 'edit-flow/edit_flow.php' );`


But It doesn't shows up in the admin-sidebar. Then I have to  use `activate_plugin(` 'edit-flow/edit_flow.php' );` to make it working as intended.

**Comment by kala725** [comment]
@david-binda  : Any update on the same.

**Comment by david-binda** [comment]
Hey @kala725 I have looked into the issues you are describing and have finally been able to reproduce the issue. Looks like the notation for loading the plugin using the `wpcom_vip_load_plugin` should be:

```
wpcom_vip_load_plugin( 'Edit-Flow/edit_flow.php' );
```

But the plugin is still missing the vipgo-helper.php file which would make sure it gets properly loaded. I have opened a PR (#390) which fixes those. Feel free to test the path using the https://github.com/Automattic/Edit-Flow/tree/fix-368/add-vipgo-helper branch
